graphql-dgs-spring-boot-micrometer/src/test/kotlin/com/netflix/graphql/dgs/metrics/micrometer/MicrometerServletSmokeTest.kt

```diff
@@ -387,7 +387,7 @@ class MicrometerServletSmokeTest {
                     .and("gql.operation.name", "anonymous")
                     .and("gql.query.complexity", "none")
                     .and("gql.query.sig.hash", "none")
-                    .and("gql.errorDetail", "none")
+                    .and("gql.errorDetail", "INVALID_ARGUMENT")
```

Comment:
Why did this change?

Reply:
explained in response to earlier comment

Reply:
The micrometer test started failing because the order in which the instrumentations run seems to have changed in my branch. If the `DgsGraphQlJavaErrorInstrumentation` runs before the `MetricsInstrumentation`, the tag value is set properly (which is in this branch). However, if it's the other way around, the metrics add the tag for gql.error before the `DgsGraphQlJavaErrorInstrumentation` runs and that's why it passes in master branch.
The correct order should be `DgsGraphQlJavaErrorInstrumentation` first and then `DgsMetricsInstrumentation` so the metrics get the correct tag value as fixed in this PR.

Basically need a way to ensure the MetricsInstrumentation runs last, but not sure what the best way to do that is, so looking into it.